addon/features.js

```diff
@@ -1,4 +1,11 @@
 export default {
+  flags: {},
+  setup: function(flags) {
+    this.flags = flags;
+  },
+  set: function(flag, enabled) {
```

Comment:
From @aexmachina: Re. set() - good point, probably enable/disable is better then. Thinking about this has made me realise that we could make flags an Ember.Object, and then people could bind {{feature.flags.foo}} in their template if they wanted to.